roles/kuryr/templates/controller-deployment.yaml.j2

```diff
@@ -49,6 +49,18 @@ spec:
           mountPath: "/etc/ssl/certs/kuryr-ca-bundle.crt"
           subPath: kuryr-ca-bundle.crt
           readOnly: true
+{% if (kuryr_controller_replicas|default(1)|int > 1) %}
+      - image: gcr.io/google_containers/leader-elector:0.5
+        name: leader-elector
+        args:
+        - "--election=kuryr-controller"
+        - "--http=0.0.0.0:${KURYR_CONTROLLER_HA_PORT:-16401}"
```

Comment:
Where is this defined? I guess we should define a kuryr default var for it

Reply:
Aww, good point, I guess I shouldn't code late in the evening.